larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -358,6 +414,49 @@ TEST_P(BConv2DOpTest, SimpleTest) {
                 float_generator);
   std::generate(std::begin(bias_data), std::end(bias_data), float_generator);
 
+  int padded_input_height = input_height;
+  int padded_input_width = input_width;
+  if (pad_values == 1) {
+    // Use a Pad op to pad with ones
+    int output_height, output_width;
+    TfLitePaddingValues padding_values = ComputePaddingHeightWidth(
+        stride_height, stride_width, dilation_height_factor,
+        dilation_width_factor, input_height, input_width, filter_height,
+        filter_width, kTfLitePaddingSame, &output_height, &output_width);
+
+    // How many pixels does the kernel stick out at the {left, top, right,
+    // bottom}
+    const int overflow_left = padding_values.width;
+    const int overflow_top = padding_values.height;
+    const int overflow_right =
+        padding_values.width + padding_values.width_offset;
+    const int overflow_bottom =
+        padding_values.height + padding_values.height_offset;
+
+    // The parameter {4,2} means that the paddings array is four pairs.
+    // The four pairs correspond to: Batch, Height, Width, Channels
+    PadOpModel<float> padop(
+        {TensorType_FLOAT32,
+         {input_batch_count, input_height, input_width, input_depth}},
+        {4, 2},
+        {0, 0, overflow_top, overflow_bottom, overflow_left, overflow_right, 0,
+         0},
+        1.0f, {TensorType_FLOAT32, {}});
+
+    padop.SetInput(input_data);
+    padop.Invoke();
+
+    padded_input_height = input_height + overflow_top + overflow_bottom;
+    padded_input_width = input_width + overflow_left + overflow_right;
+    EXPECT_THAT(padop.GetOutputShape(),
```

Comment:
here we are somehow testing the `Padding` op? if this test fails then the `one-padding` will be shown as failure but thats actually a different problem. so I think we don't need this test here?

Reply:
This is meant to verify that if the padding op somehow changes, then we will notice it here, so we will know that its not caused by our own op.